jest.setup.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const path = require('path');
+const looksSame = require('looks-same');
+const compareSvg2PngHelper = require('./test/helpers/compare-svg-2-png-helper.js');
+const capturePuppeteer = require('./test/helpers/capture-puppeteer.js');
+
+const compareSvg2PngAsync = async(receivedSVGPath, resultPNGPath, expectedPNGPath) => {
+    return new Promise((resolve, reject) => {
+        const ext = path.extname(resultPNGPath);
+        const diffPath = resultPNGPath.replace(`.${ext}`, `.diff.${ext}`);
+        compareSvg2PngHelper(receivedSVGPath, resultPNGPath, expectedPNGPath, diffPath, (err, result) => {
+            if (err) {
+                return reject(err);
+            }
+
+            resolve({ isEqual: result.equal, difference: JSON.stringify(result.diffClusters) });
+        });
+    });
+};
+
+const capturePuppeteerAsync = (previewHTML, previewImage, expectedPNGPath) => {
+    return new Promise((resolve, reject) => {
+        capturePuppeteer(previewHTML, previewImage, error => {
+            if (error) {
+                return reject(error);
+            }
+
+            looksSame(previewImage, expectedPNGPath, (error, result) => {
+                if (error) {
+                    return reject(error);
+                }
+
+                if (!result.equal) {
+                    looksSame.createDiff({
+                        reference: expectedPNGPath,
+                        current: previewImage,
+                        diff: path.join(path.dirname(previewImage), path.basename(previewImage).replace('.png', '.diff.png')),
+                        highlightColor: '#ff00ff'
+                    }, () => {});
+                }
+
+                resolve({ isEqual: result.equal, difference: JSON.stringify(result.diffClusters) });
+            });
+        });
+    });
+};
+
+// eslint-disable-next-line jest/require-hook
+expect.extend({
+    async toBeVisuallyEqual(receivedSVGPath, expectedPNGPath) {
```

Comment:
Maybe this functions should be renamed, please suggest if so.

Reply:
this one is OK. But can we move the jest files in test folder? Not sure what's the common practice here.

Reply:
@XhmikosR it can be placed inside another folder, yes. The config file may too, but in this case we need to pass it's location in script like `jest --config=path`

Reply:
Would be OK if we kept the config in root and moved the other files maybe?